starboard/android/shared/launcher.py

```diff
@@ -450,4 +450,4 @@ def GetDeviceIp(self):
 
   def GetDeviceOutputPath(self):
     """Writable path where test targets can output files"""
-    return f'/data/data/{_APP_PACKAGE_NAME}/cache/'
+    return f'/data/data/{_APP_PACKAGE_NAME}/log/'
```

Comment:
This currently controls only where the gtests Junit files get put, correct ? If so, lgtm

Reply:
I don't see any other uses for this function. I need to coordinate this with a change in Mobile Harness to merge to ensure that it knows where to look for the files.

Reply:
Ack - keep in mind "old branches" as you go through this. Although tbh we mostly care if it's correct on trunk+24 lts at this point

Reply:
I think this change should be safe to cherry pick to release branches.

I'm unresolving this conversation so this doesn't get accidentally merged before the change to MH has been lined up.

Reply:
Isn't the actual path "cache/log" ?
https://github.com/youtube/cobalt/blob/main/starboard/android/shared/system_get_path.cc#L118